style.css

```diff
@@ -130,3 +130,416 @@ p {
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  width: 247px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.work2-content li {
+  display: flex;
+  flex-direction: row;
+  align-items: right;
+  padding: 8px 10px;
+  width: 67px;
+  height: 30px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: none;
+}
+
+.work2-content ul li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  width: 60px;
+  height: 25px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.work2-button button {
+  width: 327px;
+  height: 50px;
+  color: white;
+  border: #ff6b00;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  justify-self: center;
+}
+
+.work2-button :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.about-me-1 {
+  background: url(./svg/illustration-about-me-mobile.svg);
+  background-color: #1c1a19;
+  background-position: right 0 top 400px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 340px;
+  margin-right: 1px;
+}
+
+.about-me-section {
+  background-color: #1c1a19;
+  background-position: right 0 top 400px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 211px;
+  margin: auto;
+  margin-top: 120px;
+  width: 388px;
+  width: 104%;
+}
+
+.about-me1-content {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  width: 327px;
+  align-self: center;
+}
+
+.about-me1-content > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 72px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 88px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+  color: #ff6b00;
+  margin-left: 0;
+}
+
+.about-me1-content > p {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  max-width: 100%;
+  max-height: 100%;
+  margin-top: -240px;
+  margin-left: 0;
+}
+
+.about-me-btn {
+  list-style: none;
+}
+
+.about-me-button {
+  padding: 12px;
+  width: 161px;
+  background-color: #ff6b00;
+  border: none;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 70px;
+  margin-left: 0;
+}
+
+.about-me-btn :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.line2 {
+  border: 1px solid white;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: -30px;
+}
+
+.about-me-2 {
+  width: 327px;
+  margin-top: -100px;
+  background-color: #1c1a19;
+  display: grid;
+  justify-content: center;
+  gap: 44px;
+  list-style: none;
+}
```

Comment:
- [ ] Please align your skills section properly, as I can see that your text or skills are not aligned properly with each other and looks unprofessional so Kindly fix the alignment issue to make it looks beautiful, for better understanding looks at the below comparison table.

### Design Comparison

<table>
<tr>
<th align=“middle”>Your Design</th>
<th align=“middle”>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-31 16-51-01](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/ca64eb68-c93b-4dbf-9642-4ac5941b52c4)

</td>
<td>

![Screenshot from 2023-05-31 16-51-15](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/5d8cfe4b-b28e-49b9-9f21-c5c644b46c55)

</td>
</tr>
</table>